dags/sentinel-2_data_transfer.py

```diff
@@ -89,8 +89,29 @@ def get_self_link(message_content):
         f"s3://{SRC_BUCKET_NAME}",
     )
 
+def correct_stac_link(message):
+    """
+    Replace the https link of the source bucket with s3 link of the destination bucket
+    """
 
-def publish_to_sns_topic(message):
+    body = json.loads(message.body)    
+    metadata = body.get("Message")
+
+    "Replace https with s3 uri"
+    stac = metadata.replace(
```

Comment:
Hey @omad is string.replace the right way todo this? I've been burnt by it before I think... Just going to look for an alternative.

Reply:
I'm probably being overly concerned, but in my case I was getting some gobbledigook at the end of some strings for random reasons.